### PR TITLE
WindowGUISample pull request

### PR DESCRIPTION
**Pull request by dotoritos-kim**
This example shows how to use "TrackBar" and "SetMouseCallback". Thank you for your positive review.

### CODE REVIEW

File: SamplesCS/ISample.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
+using static SamplesCS.Samples.WindowGUISample;
```
Comment (shimat) [inline SamplesCS/ISample.cs]: This seems to be unnecessary.

File: CameraOpenCV/CameraOpenCV.csproj [hunk 2]
```diff
@@ -11,8 +11,8 @@
     <AssemblyName>CameraOpenCV</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.17763.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
```
Comment (shimat) [inline CameraOpenCV/CameraOpenCV.csproj:15]: 🤔

File: SamplesCS/ISample.cs (hunk 1 above)
Comment (dotoritos-kim) [reply SamplesCS/ISample.cs]: I have accepted your advice. Thank you

File: SamplesCS/Samples/PerspectiveTransformSample.cs [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        }
+
+        private void CallbackOpenCVAnnotate(MouseEvent e, int x, int y, MouseEvent flags, IntPtr userdata)
+        {
+            if (e == MouseEvent.LButtonDown)
+            {
+                point2Fs.Add(new Point2f(x, y));
+                if (point2Fs.Count == 4)
+                {
+                    srcPoints = new Point2f[]
```
Comment (shimat) [inline SamplesCS/Samples/PerspectiveTransformSample.cs]: `srcPoints = point2Fs.ToArray();`

File: SamplesCS/Samples/WindowGUISample.cs [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+            var erodeDilateWindow = new Window("Erode/Dilate", image: dst);
+            var erodeDilateTrackbar = erodeDilateWindow.CreateTrackbar(
+                trackbarName: "Iterations", 
+                value: 10, 
+                max: maxIterations * 2 + 1,
+                callback: (pos, obj) =>
+                {
+                    var n = pos - maxIterations;
+                    var an = n > 0 ? n : -n;
+                    var element = Cv2.GetStructuringElement(
```
Comment (shimat) [inline SamplesCS/Samples/WindowGUISample.cs]: using var element = ...

File: SamplesCS/Samples/PerspectiveTransformSample.cs (hunk 3 above)
Comment (dotoritos-kim) [reply SamplesCS/Samples/PerspectiveTransformSample.cs]: Thank you for not thinking

**Review by shimat — APPROVED** [review]
Thanks!

**Pull request merged**